notification/tests/test_tasks.py

```diff
@@ -0,0 +1,157 @@
+from datetime import datetime, timedelta
+
+import pytest
+
+from notification import tasks
+from notification.models import NotificationSetting, Notification
+from test.factories import *
+from django.test import override_settings
+
+
```

Comment:
I would add one additional test to check that when `only_one_day` is True, the notifications from the previous days are not processed.

Reply:
done